Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+ruby "2.4.1"
```

Comment:
@Rojo Maybe you didn't notice, 2.4 has major changes from the previous versions. For example, it deprecated `fixnum`. Those major changes causes programs written in the previous versions to break. I already encountered it. 

Also, I just deployed an app to heroku yesterday. For app written in 2.3.1, the version heroku applies to it if no version specified in `Gemfile` is 2.3.4. That implies the problem I mentioned above.